File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>
//define function that prints the bricks
void bricks(int n)
{
    for (int i = 0; i < n; i++)
    {
        printf("#");
    }
}

//define function that prints the spaces
void spaces(int m)
{
    for (int j = 0; j < m; j++)
    {
        printf(" ");
    }
}

//main function
int main(void)
{
    int h;
    //user input number between 1-8 for height, repeat till user complies
    do
    {
        h = get_int("Height: ");
    }
    while (((h < 1) || (h > 8)));
    
    int x = 1;
    int y = 1;
    //repeat function till it prints required number of bricks
    while ((y <= h) && (x <= h))
    {
        spaces(h - y);
        y++;
        bricks(h - (h - x));
        x++;
        printf("\n");
    }
}

File: sentimental-mario-less/mario.py
# import get_int from cs50 library
from cs50 import get_int

# define main


def main():
    # get user input by calling get_height
    height = get_height()

    # loop printing accordign to entered height
    for i in range(height):
        bricks = i + 1
        spaces = height - 1
        print(" " * spaces + "#" * bricks)
        height -= 1
        bricks += 1

# defining get_height


def get_height():
    # do while loop till user complies
    while True:
        try:
            height = get_int("Height: ")
            if height > 0 and height < 9:
                break
        except:
            print("Enter a valid number")
    return height

main()

